test/test_examples.jl

```diff
@@ -2,29 +2,46 @@ const EXAMPLES_DIR = "../examples/"
 
 function run_example(replace_strings, example_name, module_suffix="")
     example_filepath = joinpath(EXAMPLES_DIR, example_name * ".jl")
-    txt = read(example_filepath, String)
+    file_content = read(example_filepath, String)
 
     for strs in replace_strings
-        txt = replace(txt, strs[1] => strs[2])
+        file_content = replace(file_content, strs[1] => strs[2])
     end
 
     test_script_filepath = example_name * "_example_test.jl"
 
     open(test_script_filepath, "w") do f
         write(f, "module Test_$example_name" * "_$module_suffix\n")
-        write(f, txt)
+        write(f, file_content)
         write(f, "\nend # module")
     end
 
     try
         include(test_script_filepath)
     catch err
+        # Throw the error
         @error sprint(showerror, err)
+
+        # Print the content of the file to the test log, with line numbers, for debugging
+        delineated_file_content = split(file_content, '\n')
+
+        @printf "% 3d module Test_%s_%s\n" 1 example_name module_suffix
+
+        for (line, number) in enumerate(delineated_file_content)
```

Comment:
I think you got it the other way around?
```suggestion
        for (number, line) in enumerate(delineated_file_content)
```

Reply:
Thank you! Yeah, we don't exactly have tests for tests, and this part of the code only runs when there's an error. We'll probably see the error sooner or later...